pkg/jobs/adopt.go

```diff
@@ -82,7 +82,7 @@ func (r *Registry) maybeDumpTrace(resumerCtx context.Context, resumer Resumer, j
 		return
 	}
 
-	resumerTraceFilename := fmt.Sprintf("resumer-trace-n%s.%s.txt",
+	resumerTraceFilename := fmt.Sprintf("resumer-trace-n%s.%s.protobin.txt",
```

Comment:
this isn't _quite_ what I was suggesting: I was thinking the filename written here would just be ".protobin" (actually I just checked the docs and they say to use `.binpb`), so you could download the raw proto from the artifacts page if you wanted to. But during listing, if we see a a file where `HasSuffix(".binpb")`, we _synthesize_ an additional entry in the returned list that has a `.binpb.txt` suffix, which we use on the read path to know to go fetch the other file from info and string it?

Reply:
Eventually maybe we stop injecting them into the list, and teach the frontend to just have two download links on them -- `view text`, `download raw` -- but initially it seems easiest just to fake out an entry in the artifact list api

Reply:
actually i noticed a bug when playing around with the download functionality.

Reply:
should be good now